kfdefs/overlays/moc/smaug/opf-jupyterhub/pvcs/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: opf-jupyterhub-stage
```

Comment:
you can remove this namespace. This will be applied by the opf-jupyterhub kustomization